### DISCUSSION TITLE
Export list of connections/contacts for syncing between computers

### DISCUSSION THREAD

**Discussion by Oceanwatcher** [rustdesk/rustdesk#2854] [General]
I work on two different computers and would like to have the same list of contacts/computers in Rustdesk on both computers.

The ideal things would be to be able to sync a file with this connection info between the computers using Syncthing or similar. Is the configuration stored in such a way that it would be possible?

**Comment by rustdesk** [comment]
`C:\Users\<User>\AppData\Roaming\Rustdesk\config\peers`

**Reply by Oceanwatcher** [reply]
Thank you very much! Very useful!

**Reply by rakleed** [reply]
for macOS: `~/Library/Preferences/com.carriez.RustDesk/peers`

**Reply by Oceanwatcher** [reply]
I was not able to find that on my Mac (10.15).

**Reply by rakleed** [reply]
@Oceanwatcher you searched exactly in the user's directory? Try copying my path, open Finder, press Cmd+Shift+G, paste this path and click Enter.

And I'm using version 1.2.0 nightly.

**Reply by Oceanwatcher** [reply]
I actually searched my system for the term peers. Only found the file I sent over from my Win computer...

**Comment by Sikkim76** [comment]
Hello. 
I have a big request for you. When I reinstalled my computer, I copied the peers directory from the config folder with contacts to my usb drive. When I reinstalled the computer and reinstalled rustdesk, I copied the peers directory back to the config folder. But my contacts did not appear after booting. I have about 30 contacts in there and I don't want to manually overwrite it. Please is there a guide to transfer contacts between PCs or when reinstalling? I couldn't find this topic anywhere. 
Thank you very much. 

Martin

**Reply by Avrution** [reply]
Same issue.  Moved the peers folder, but nothing updates on the other machine.

**Comment by AgentEddie99** [comment]
I personally use RDM (Remote Desktop Manager) and have the connections synced in my OneDrive folder, then setup RDM to have the connections there. RDP, SSH and RustDesk.

Example of my RDM. 
<img width="284" height="290" alt="image" src="https://github.com/user-attachments/assets/010d1fa8-96d4-408b-ad50-d4b4ef90b12d" />
